### DISCUSSION TITLE
Blu-ray Backups Only (No Transcode)

### DISCUSSION THREAD

**Discussion by BryanGoble** [automatic-ripping-machine/automatic-ripping-machine#1268] [General]
Is there a setting in ARM that I can set to just have MakeMKV/Handbrake rip a Blu-ray without transcoding? I'm on a less powerful machine right now and would like to rip the raw files only. I'll then transcode those later manually when I'm back at my much more powerful desktop

**Comment by mihawk90** [comment]
You should probably take a look at the [arm.yaml](https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/config-arm.yaml), it's all in there.

**Comment by BryanGoble** [comment]
> You should probably take a look at the [arm.yaml](https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/config-arm.yaml), it's all in there.

I did, but it isn't clear. That's why I asked the question. The RIP method variable only discusses how it's ripped but doesn't clear state if transcoding is skipped. All I need to know is if it can be skipped and what the appropriate setting would be.

**Reply by microtechno9000** [reply]
Setting Transcode skip to true will do it

**Reply by mihawk90** [reply]
> doesn't clear state if transcoding is skipped

That's what `SKIP_TRANSCODE` is for.

**Reply by BryanGoble** [reply]
> > doesn't clear state if transcoding is skipped
> 
> That's what `SKIP_TRANSCODE` is for.

Thanks, I should've been more clear though. Am I able to only skip transcoding for Blu-rays while continuing with the whole process for DVDs or would be better to just spin up a separate docker container for Blu-rays?

**Reply by mihawk90** [reply]
As far as I know there are no separate settings for that. You can either go the 2-instances route you're suggesting or see if you can get a Handbrake profile to work where it just skips files larger then DVD resolution.

**Reply by nirvdrum** [reply]
Should `SKIP_TRANSCODE` work when the ripping method is set to `backup`? The docs suggest that option is only honored when the ripping method is set to `mkv`, but the answer to OP's question suggests that it should work for backups as well. I've set `SKIP_TRANSCODE=True` and `RIPMETHOD=backup`, but I'm seeing `HandBrakeCLI` running after the rip completes. I'm trying to sort out whether that's a limitation of the software, a bad configuration on my end, or a problem with the process loading the config file.
